### ISSUE TITLE
Rendering issue with `infix` notation

### ISSUE THREAD

**Issue by pkra** [mathjax/MathJax-node#66]
Sample via Benetech

```html
<math xmlns:mml="http://www.w3.org/1998/Math/MathML" mode="display" overflow="scroll">
          <mml:mrow>
            <mml:mfrac>
              <mml:mrow>
                <mml:mi>d</mml:mi>
 
                <mml:mi>x</mml:mi>
              </mml:mrow>
 
              <mml:mrow>
                <mml:mi>d</mml:mi>
 
                <mml:mi>θ</mml:mi>
              </mml:mrow>
            </mml:mfrac>
 
            <mml:mo>=</mml:mo>
 
            <mml:mfrac>
              <mml:mi>β</mml:mi>
 
              <mml:mrow>
                <mml:msup>
                  <mml:mo form="prefix">cos</mml:mo>
 
                  <mml:mn>2</mml:mn>
                </mml:msup>
 
                <mml:mi>θ</mml:mi>
              </mml:mrow>
            </mml:mfrac>
          </mml:mrow>
        </mml:math>
```

Sample Rendering: 

![unnamed](https://cloud.githubusercontent.com/assets/426686/6017012/a457b98e-ab89-11e4-9039-1255e970ca32.jpg)

Since this doesn't happen on in core MathJax, it seems like a mathjax-node bug.

**Comment by pkra** [comment]
Should be fixed after https://github.com/mathjax/MathJax/pull/1040

**Comment by pkra** [comment]
Fixed after update to MathJax 2.5.

**Comment by pkra** [comment]
I tried this on a fresh develop branch and now get an error :-(

```
file:///tmp/MathJax-node/node_modules/MathJax/unpacked/MathJax.js:2353
        error.oncontextmenu = MathJax.Extension.MathEvents.Event.Menu;
                                                          ^
TypeError: Cannot read property 'Event' of undefined
    at file:///tmp/MathJax-node/node_modules/MathJax/unpacked/MathJax.js:2353:59
    at Function.CALLBACK.execute (file:///tmp/MathJax-node/node_modules/MathJax/unpacked/MathJax.js:236:26)
    at cb (file:///tmp/MathJax-node/node_modules/MathJax/unpacked/MathJax.js:218:59)
    at Object.BASE.Ajax.Require (file:///tmp/MathJax-node/node_modules/MathJax/unpacked/MathJax.js:725:9)
    at Object.MathJax.Hub.formatError (file:///tmp/MathJax-node/node_modules/MathJax/unpacked/MathJax.js:2352:20)
    at Object.MathJax.Hub.processError (file:///tmp/MathJax-node/node_modules/MathJax/unpacked/MathJax.js:2322:12)
    at Object.MathJax.Hub.processOutput (file:///tmp/MathJax-node/node_modules/MathJax/unpacked/MathJax.js:2300:32)
    at Function.CALLBACK.execute (file:///tmp/MathJax-node/node_modules/MathJax/unpacked/MathJax.js:236:26)
    at cb (file:///tmp/MathJax-node/node_modules/MathJax/unpacked/MathJax.js:218:59)
    at BASE.Object.Subclass.Process (file:///tmp/MathJax-node/node_modules/MathJax/unpacked/MathJax.js:488:38)
```

**Comment by pkra** [comment]
closing this again in favor of #85

**Comment by dpvc** [comment]
This should be resolved by commit 0b4b10f listed above.
